app/generators/question_type_generator.rb

```diff
@@ -17,7 +18,12 @@ def generate_tooltip(tooltip_content)
     tooltip_body = content_tag(:i, 'info', class: 'tooltip flow-text material-icons info-outline')
     content_tag(:a,
                 tooltip_body,
-                onclick: "M.toast({html: '#{tooltip_content.gsub("'", %q(\\\'))}'}, #{TOOLTIP_DURATION})")
+                onclick: "M.toast({html: '#{tooltip_content.gsub("'", %q(\\\'))}'," \
+                         " displayLength: #{tooltip_duration(tooltip_content)}})")
+  end
+
+  def tooltip_duration(tooltip_content)
```

Comment:
haha fancy. dit kunnen we weer machine learnen en kijken hoelang mensen er over doen om een karakter te lezen

Reply:
Trouwens, ik kan me herinneren dat je ook een raw had met een i'tje. Heb je die ook aangepast / kan die ook gebruik maken van deze functionaliteit?

Reply:
Ja ik heb de duration van die raw met het i'tje erin ook berekend via deze formule.

En ik heb een hele safe margin genomen nu. Je kan de tekst rustig twee keer lezen in de tijd die ervoor beschikbaar is. Ook omdat het makkelijker is weg te klikken dan het irritant is dat dat ding weggaat als je nog bezig bent met lezen.